demo/csrc/cpp/restorer.cxx

```diff
@@ -2,33 +2,39 @@
 
 #include "mmdeploy/restorer.hpp"
 
-#include <opencv2/imgcodecs/imgcodecs.hpp>
-#include <opencv2/imgproc/imgproc.hpp>
-#include <string>
+#include "opencv2/imgcodecs/imgcodecs.hpp"
+#include "opencv2/imgproc/imgproc.hpp"
+#include "utils/argparse.h"
+
+DEFINE_ARG_string(model, "Super-resolution model path");
+DEFINE_ARG_string(image, "Input image path");
+DEFINE_string(device, "cpu", R"(Device name, e.g. "cpu", "cuda")");
+DEFINE_string(output, "restorer_output.jpg", "Output image path");
```

Comment:
".bmp" is expected.

Reply:
Why do you think ".bmp" is expected? BMP is extremely large and everywhere in the rest of the whole project is using ".jpg".